edward/util/metrics.py

```diff
@@ -0,0 +1,47 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+
+from functools import wraps
+
+import tensorflow as tf
+
+
+def with_binary_averaging(metric):
+  """
+  Inspired by scikit-learn's _average_binary_score function:
+  http://bit.ly/2yhcABp.
```

Comment:
Short links are bad in docstrings because they're not transparent about the url. Can you paste the full link instead?